src/Microsoft.DotNet.Build.Tasks/PackageFiles/CodeCoverage.targets

```diff
@@ -6,7 +6,7 @@
   -->
   <PropertyGroup>
     <OpenCoverVersion>4.6.519</OpenCoverVersion>
-    <ReportGeneratorVersion>2.5.0</ReportGeneratorVersion>
+    <ReportGeneratorVersion>3.0.1</ReportGeneratorVersion>
```

Comment:
The ReportGenerator version is 3.0.1 in the CoreFX version of this.  I am not sure why that is (does something update this file as it gets installed into CoreFx\Tools?) but it seems better to move forward since CoreFX is the major (only?) client of code coverage.